doc/turn-off.rst

```diff
@@ -1,9 +1,26 @@
 Tear down your Binder deployment
---------------------------------
+================================
 
 Deconstructing a Binder deployment can be a little bit confusing because
 users may have caused new cloud containers to be created. It is important
 to remember to delete each of these containers or else they will continue
 to exist (and cost money!).
 
-<<<TODO>>>
+Contracting the size of your cluster
+------------------------------------
+
+If you would like to shrink the size of your cluster, refer to the
+`Expanding and contracting the size of your cluster <https://zero-to-jupyterhub.readthedocs.io/en/latest/extending-jupyterhub.html#expanding-and-contracting-the-size-of-your-cluster>`_
+section of the Zero to Jupyter documentation. Resizing the cluster to zero nodes
```

Comment:
This sounds reasonable to me, but I'd like to hear @yuvipanda 's opinion on this too...we're not unintentionally asking people do to something they'd regret later by doing this, are we?

Reply:
Yes, resizing to zero is safe. It doesn't delete the master node, which is managed separately by the cloud provider, and doesn't delete volumes or deployments.